src/domain/registration/state/RegistrationSelectors.tsx

```diff
@@ -0,0 +1,4 @@
+import { StoreState } from '../../app/types/stateTypes';
```

Comment:
Wouldn't `.ts` be the correct file-extension for this file?

Reply:
Agree

Reply:
Would reducer should be `.ts` too?

Reply:
I think it should only be tsx/jsx when the file has some jsx in it, so usually just components (or their tests)